bsd/api.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+import os
+from bsdapi.BsdApi import Factory as BsdApiFactory
+from django.conf import settings
+
+
+class Bsd:
```

Comment:
Technically PEP8 also recommends naming this `BSD`, but up to you because it's a heated debate and I see BSD API doesn't follow this convention apparently. 

https://www.python.org/dev/peps/pep-0008/#id36
```Note: When using abbreviations in CapWords, capitalize all the letters of the abbreviation. Thus HTTPServerError is better than HttpServerError.```